about.html

```diff
@@ -0,0 +1,123 @@
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+          </div>
+        <header>
+          <div id="drop-menu">
+            <img src="Images/Icons/humberge.svg" alt="show menu" />
+          </div>
+          <div id="logo">
+            <a href="./"><img src="./Images/Icons/logo.svg" alt="" /></a>
+          </div>
+          <nav class="hide">
+            <div id="close-menu">
+              <img src="./Images/Icons/delete.svg" alt="close menu" />
+            </div>
+            <ul>
+              <li>
+                <a href="./">Home</a>
+              </li>
+              <li>
+                <a href="about.html">About</a>
+              </li>
+              <li>
+                <a href="#main-tutors">Classes</a>
+              </li>
+              <li>
+                <a href="#customers">Reputation</a>
+              </li>
+              <li>
+                <a href="#partners">Sponsor</a>
+              </li>
+              <li>
+                <a href="#">News</a>
+              </li>
+              <li>
+                <a href="#">WM campaign</a>
+              </li>
+            </ul>
+            <div class="remaining"></div>
+          </nav>
+        </header>
+      <section id="about-us">
+        <p class="title-3">"Hello! Website developers"</p>
+        <h1 class="title-1">WEBSITE DEVELOPMENT <br/> AWESOME TUTORIALS 2022</h1>
+        <div class="card">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin posuere vestibulum est in faucibus. Aenean molestie nisl in magna consectetur, placerat consequat nulla euismod. Vivamus eleifend est blandit dictum molestie.</p>
+        
+
+          </div>          
+          <p>Please contact us per Email for any further questions about Our web-mastery class, 2022!<p>
+          <p class="email">Tadalem26@gmail.com</p>
+      </section>
+    </main>
+    <section id="more-about">
+      <h2 class="title-2">Creative design and mastery of coding. By Tadesse Alemayehu</h2>
+      <hr class="short">
+    <figure class="logo-card">
+      <img src="./Images/Icons/logo.svg" alt="WinMac web development">
+      <figcaption>WinMac web-dev 2022</figcaption>
+    </figure>
+    </section>
+      <hr class="full">
+      <section id="past-graduates">
+<h2 class="title-2">See the past WM graduates.</h2>
+<hr class="short">
+<p> See our graduate students, hear what they have to say about us, it will all make sense.</p>
+<div id="graduates">
+  <div class="container">
+      
+    <div class="content"><div class="text">
+      <h1 class="title-1">2021</h1>
+      <p>WinMac Web development school 2022</p></div>
+    </div>
+    
+  </div>
+  <div class="container">
+    <div class="content"><div class="text">
+      <h1 class="title-1">2020</h1>
+      <p>WinMac Web development school 2022</p></div>
+    </div>
+      
+    </div>
+  </div>
+</div>
+      </section>
+    <footer>
```

Comment:
The footer should have a dark background on your about page in desktop sizes 📝 Please make the necessary changes so that your project meets the requirements 👌

Reply:
@mateo951 Thank you ✋. ill make the necessary changes right away.